more_autograding_examples/postgres_database_queries/config.json

```diff
@@ -0,0 +1,54 @@
+{
+  "resource_limits": {
+    "RLIMIT_CPU": 180
+  },
+  "autograding_method": "docker",
+  "container_options": {
+    "use_router": false
+  },
+  "autograding": {
+    "submission_to_runner": ["**/*.py", "**/*.txt","**/*.sql"],
+    "work_to_details": ["**/*.txt"]
+  },
+  "testcases": [
+    {
+      "title": "Queries",
+      "points": 15,
+      "validation": [
+        {
+          "method": "diff",
+          "description": "One",
+          "deduction": 0.1,
+          "actual_file": "main/1-result.txt",
+          "expected_file": "1-expected.txt"
+        },
+        {
+          "method": "diff",
+          "description": "Two",
+          "deduction": 0.1,
+          "actual_file": "main/2-result.txt",
+          "expected_file": "2-expected.txt"
+        },
+        {
+          "method": "diff",
+          "description": "Three",
+          "deduction": 0.1,
+          "actual_file": "main/3-result.txt",
+          "expected_file": "3-expected.txt"
+        }
+      ],
+      "containers": [
+        {
+          "server": true,
+          "container_name": "db_example_postgres",
+          "container_image": "goofykubers/example_db_autograding:latest"
```

Comment:
This is intended to be the Dockerfile below? Can we just change the name to be `example_db:latest` or something and not a real image that can be found on https://hub.docker.com unless the intention was to publish this image on submittyrpi as well. I'd rather not include "random" users in examples (even though in this instance it's Prof. Johnson).

Reply:
As in regards to the name sure thing. 

As discussed in our meeting today @emaicus plans on publishing this example db image under submitty dockerhub account.